api/base/settings/__init__.py

```diff
@@ -11,17 +11,18 @@
 import itertools
 
 from .defaults import *  # noqa
+from website import settings
 
 try:
     from .local import *  # noqa
 except ImportError as error:
     warnings.warn('No api/base/settings/local.py settings file found. Did you remember to '
                   'copy local-dist.py to local.py?', ImportWarning)
 
-if not DEBUG and os.environ.get('DJANGO_SETTINGS_MODULE') == 'api.base.settings':
+if not settings.DEV_MODE and os.environ.get('DJANGO_SETTINGS_MODULE') == 'api.base.settings':
```

Comment:
Minor: `DEV_MODE` appears to be the only thing all of `website.settings` is imported for.